### PR TITLE
CB-3311 feat: application update message

### PR DESCRIPTION
**Pull request by Wroud**

### CODE REVIEW

File: webapp/packages/core-cli/configs/webpack.config.js [hunk 1]
```diff
@@ -106,6 +107,25 @@ module.exports = (env, argv) => {
...
+    // workboxPlugin = new WorkboxPlugin.InjectManifest({
+    //   swSrc: getServiceWorkerSource(),
+    //   swDest: 'service-worker.js',
+    // });
+    // Object.defineProperty(workboxPlugin, 'alreadyCalled', {
+    //   get() {
+    //     return false;
+    //   },
+    //   set() {},
+    // });
```
Comment (tfokina) [inline webapp/packages/core-cli/configs/webpack.config.js:123]: Are those comments necessary here? If they are, maybe it's good to add some TODO message?

File: webapp/packages/core-localization/src/locales/ru.ts [hunk 2]
```diff
@@ -99,7 +99,7 @@ export default [
   ['app_root_session_expire_warning_button', 'Я тут'],
   ['app_root_session_expired_title', 'Сессия истекла'],
   ['app_root_session_expired_message', 'Сессия истекла. Хотите перезагрузить страницу?'],
-  ['app_root_session_expired_reload', 'Перезагрузить'],
+  ['ui_processing_reload', 'Перезагрузить'],
```
Comment (tfokina) [inline webapp/packages/core-localization/src/locales/ru.ts]: Line 11 duplicate?

Comment (Wroud) [reply webapp/packages/core-localization/src/locales/ru.ts]: removed

File: webapp/packages/core-cli/configs/webpack.config.js (hunk 1 above)
Comment (Wroud) [reply webapp/packages/core-cli/configs/webpack.config.js:123]: comment added

**Review by tfokina — APPROVED** [review]

**Pull request merged**